packages/esm-patient-common-lib/src/workspaces/workspaces.ts

```diff
@@ -33,7 +33,7 @@ export interface WorkspaceRegistration {
   /** Use `getLifecycle` or `getAsyncLifecycle` to get the value of `load` */
   load(): Promise<any>;
   /** Only one of each "type" of workspace is allowed to be open at a time. The default is "form" */
-  type?: string;
+  type?: 'clinical-form' | 'visit-note' | 'order' | string;
```

Comment:
Does having specific types help with anything here?

Reply:
It should either have the specific constants or `string`, not both. I originally made it "string" so that if third-party modules wanted to add their own workspace types and buttons to the siderail they could do that. Limiting the allowed values of "type" might help us catch invalid type errors in our code a little bit easier, but would make it harder for third-party modules using Typescript to have their own workspace types. So this is a judgement call based on whether we want to support third party modules being able to do that. I suspect that we do.

Reply:
I have reverted the type to just string, I agree, that might have bring in a lot of issues.
Having it string, and making it default to `form` is okay I believe, because in the community version, the main distinction is actually between siderail workspaces and other workspaces.

If we consider the third-party modules, yes it will also conflict there as well.
Thanks for the help @brandones @ibacher !